persistence-api/src/main/java/io/stargate/db/DbActivator.java

```diff
@@ -0,0 +1,71 @@
+package io.stargate.db;
+
+import com.codahale.metrics.MetricRegistry;
+import io.stargate.db.cdc.CDCProducer;
+import io.stargate.db.cdc.CDCService;
+import io.stargate.db.cdc.CDCServiceImpl;
+import org.osgi.framework.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DbActivator implements BundleActivator, ServiceListener {
```

Comment:
Question for the future, how it will look like if there will be > 1 underlying `CDCProducers` that we need to support and register (i.e. kafka and kinesis).

Reply:
I think there should be only 1 cdc producer. CDC is part of the hot write path and it fails the request when the producer fails, so there shouldn't be multiple producers.

I think there's the idea of multiple streams integrations, like push this a stream A, push this to stream B, but its not necessarily related to CDC. We may share the same persistence primitives but not much (i.e. it will not block the write request).